spec/core_functions/string/split.hrx

```diff
@@ -0,0 +1,225 @@
+<===> limit/input.scss
+@use "sass:string";
+a {b: string.split("a, b, c, d", ", ", 2)}
+
+<===> limit/output.css
+a {
+  b: ["a", "b", "c, d"];
+}
+
+<===>
+================================================================================
+<===> double_width_character/input.scss
+@use "sass:string";
+// Sass treats strings as sequences of Unicode codepoint; it doesn't care if a
+// character is represented as two UTF-16 code units.
+a {b: string.split("👭a", "")}
+
+<===> double_width_character/output.css
+a {
+  b: ["👭", "a"];
+}
+
+<===>
+================================================================================
+<===> private_use_character/input.scss
+@use "sass:string";
+// Dart Sass emits private-use characters as escapes in expanded mode, but it
+// should still treat them as single characters for the purpose of functions.
+a {b: string.split("\E000", "")}
+
+<===> private_use_character/output.css
+a {
+  b: ["\E000"];
+}
+
+<===>
+================================================================================
+<===> named/input.scss
+@use "sass:string";
+a {b: string.split($string: "a/b/c", $separator: "/", $limit: 1)}
+
+<===> named/output.css
+a {
+  b: ["a", "b/c"];
+}
+
+<===>
+================================================================================
+<===> error/too_many_args/input.scss
+@use "sass:string";
+a {b: string.split("a/b/c", "/", 1, 3)}
+
+<===> error/too_many_args/error
+Error: Only 3 arguments allowed, but 4 were passed.
+1 | a {b: string.split("a/b/c", "/", 1, 3)}
+  |       ^^^^^^^^^^^^^^^^^^^^^^^^^ invocation
+  '
+  ,--> sass:string
+1 | @function split($string, $separator, $limit: null) {
+  |           ======================================== declaration
+  '
+  input.scss 1:7  root stylesheet
+
+<===>
+================================================================================
+<===> error/too_few_args/input.scss
+@use "sass:string";
+a {b: string.split("a/b/c")}
+
+<===> error/too_few_args/error
+Error: Missing argument $separator.
+1 | a {b: string.split("a/b/c")}
+  |       ^^^^^^^^^^^^^^^^^^^^^^^^^ invocation
+  '
+  ,--> sass:string
+1 | @function split($string, $separator, $limit: null) {
+  |           ======================================== declaration
+  '
+  input.scss 1:7  root stylesheet
+
+<===>
+================================================================================
+<===> error/type/string/input.scss
+@use "sass:string";
+a {b: string.split(1, "%")}
+
+<===> error/type/string/error
+Error: $string: 1 is not a string.
+  ,
+1 | a {b: string.split(1, "%")}
+  |       ^^^^^^^^^^^^^
+  '
+  input.scss 1:7  root stylesheet
+
+<===>
+================================================================================
+<===> error/type/separator/input.scss
+@use "sass:string";
+a {b: string.split("1/2/3", /)}
+
+<===> error/type/separator/error
+Error: $separator: / is not a string.
+  ,
+1 | a {b: string.split("1/2/3", /)}
```

Comment:
This is a syntax error, since `/` isn't a valid Sass expression.

Reply:
Fixed. Changed this separator to a number.